src/main/resources/config/application.yml

```diff
@@ -117,7 +118,9 @@ artemis:
             short-name: "artemis-build-agent-1"
             display-name: "Artemis Build Agent 1"
 
-
+    version-control:
```

Comment:
When removing the buildagent profile/yaml, we need to set those as defaults here